src/globus_sdk/utils.py

```diff
@@ -1,13 +1,17 @@
 import hashlib
 from base64 import b64encode
 from collections import UserDict
+from typing import Optional
 
+from .types import ToStr
 
-def sha256_string(s):
+
+def sha256_string(s: str) -> str:
     return hashlib.sha256(s.encode("utf-8")).hexdigest()
 
 
-def safe_b64encode(s):
+# REVIEW
+def safe_b64encode(s) -> str:  # type: ignore
```

Comment:
I think the intent -- based on the function body -- is `s: Union[bytes, str]`.

Reply:
I accidentally resolved this at one point.
Needs the adjustment + removal of `REVIEW` comment.